src/sphinxtheme/plone/plone_basic/layout.html

```diff
@@ -295,16 +295,35 @@
 </script>
 {%- endif %}
 
-<!-- Below block includes our gitter widget -->
-{%- if theme_use_gitter %}
-<script>
-  ((window.gitter = {}).chat = {}).options = {
-    room: 'plone/public'
-  };
-</script>
-<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer>
-</script>
-{%- endif %}
+<!-- Below block includes our discord widget -->
+<style>
+  .floating-discord-btn{
+    width: 150px;
+    height: 40px;
+    background: #00a080cb;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    color: white;
+    border-radius: 2.5%;
+    font-size: 12px;
+    position: fixed;
+    right: 30px;
+    bottom: 0px;
+
+    outline: turquoise;
+    border: none;
+    cursor: pointer;
+  }
+
+</style>
```

Comment:
Let's make this a little more consistent with the existing button.

Also it should go into the style sheet, not inline in the template.

https://github.com/plone/sphinxtheme.plone/blob/a0ea9d4a1b7f87f5bb26efa828b70fd15f6d4f30/src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css#L100

```css
a.floating-discord-btn {
    background: #00a080cb;
    border-top-left-radius: 0.5em;
    border-top-right-radius: 0.5em;
    bottom: 0px;
    color: #fff;
    font-size: 12px;
    letter-spacing: 1px;
    padding: 1em 3em;
    position: fixed;
    right: 10px;
    text-decoration: none;
}
```

Reply:
Okay... I'll do this in comparison to our current button....

Reply:
Not applying the exact changes as declared in the stylesheet! @stevepiercy

Reply:
@stevepiercy please have a look at last commit... Is that working?